lib/crowbar/client/request/backup/upload.rb

```diff
@@ -61,12 +61,21 @@ def method
           # @return [String] path to the API endpoint
           #
           def url
-            [
-              "api",
-              "crowbar",
-              "backups",
-              "upload"
-            ].join("/")
+            case Config.apiversion
+            when 1.0
+              [
+                "utils",
+                "backups",
+                "upload"
+              ].join("/")
+            when 2.0
+              [
+                "api",
+                "crowbar",
+                "backups",
+                "upload"
+              ].join("/")
+            end
```

Comment:
I prefer having the join after the end. Then we can remove it from the arrays...